editor/src/components/canvas/remix/remix-error-handling.spec.tsx

```diff
@@ -1,14 +1,21 @@
-import { setFeatureForUnitTestsUseInDescribeBlockOnly } from '../../../utils/utils.test-utils'
+import { makeTestProjectCodeWithSnippet, renderTestEditorWithCode } from '../ui-jsx.test-utils'
 import { runTestReturningErrorBoundaries } from './remix-error-handling.test-utils'
 
-// This test and its sibling have to be kept in separate files,
-// as otherwise it causes issues with Jest for some reason
-
 describe('Remix error handling', () => {
+  it('Supports a user-defined error boundary', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(`<div>Helo</div>`),
+      'await-first-dom-report',
+    )
+
+    expect(renderResult.renderedDOM.getByText('Helo')).toBeDefined()
```

Comment:
What's going on with this test? It's not at all the original test...

Reply:
OH HAHAHA JESUS